src/lib.rs

```diff
@@ -165,6 +165,8 @@ impl CertPaths {
             {
                 Ok(certs) => certs,
                 Err(err) => {
+                    #[cfg(feature = "tracing")]
+                    tracing::error!("error loading certs: {:?}", err);
```

Comment:
"from file"?

Reply:
this is actually already decorated into the underlying error. sample output:

```
2024-08-27T09:10:37.040117Z ERROR rustls_native_certs: error loading certs: Custom { kind: NotADirectory, error: "could not load certs from dir /dev/null: Not a directory (os error 20)" }
```